src/main/java/org/jvnet/hudson/plugins/thinbackup/ThinBackupMgmtLink.java

```diff
@@ -184,12 +192,135 @@ public void doSaveSettings(
     }
 
     public ThinBackupPluginImpl getConfiguration() {
-        return ThinBackupPluginImpl.getInstance();
+        return ThinBackupPluginImpl.get();
     }
 
-    public List<String> getAvailableBackups() {
-        final ThinBackupPluginImpl plugin = ThinBackupPluginImpl.getInstance();
-        return Utils.getBackupsAsDates(new File(plugin.getExpandedBackupPath()));
```

Comment:
Was the `getAvailableBackups()` method deleted in error? It seems to be still used in `src/main/resources/org/jvnet/hudson/plugins/thinbackup/ThinBackupMgmtLink/restoreOptions.jelly`:

```xml
<j:set var="allBackups" value="${it.availableBackups}"/>
```

If it is indeed still in use, then we can simply restore these deleted lines.

Reply:
I will restore it, unfortunatly, the `restoreOptions.jelly` does not work anymore, it dows not fill the available backups. So I tried something with the `ListBoxModel` but also did not succeed yet.